frontend/package.json

```diff
@@ -7,24 +7,8 @@
     "lib"
   ],
   "scripts": {
-    "postinstall": "patch-package",
     "start": "yarn workspace @streamlit/app start",
-    "build": "yarn workspace @streamlit/lib build && yarn workspace @streamlit/app build",
-    "buildFast": "yarn workspace @streamlit/app buildFast",
-    "buildLib": "yarn workspace @streamlit/lib build",
-    "buildLibProd": "yarn workspace @streamlit/lib build:prod",
-    "buildApp": "yarn workspace @streamlit/app build",
-    "typecheck": "yarn workspace @streamlit/lib typecheck && yarn workspace @streamlit/app typecheck",
-    "lint": "yarn lintApp && yarn lintLib",
-    "lintApp": "yarn workspace @streamlit/app lint",
-    "lintLib": "yarn workspace @streamlit/lib lint",
-    "format": "prettier --write --config .prettierrc './{app,lib}/src/**/*.{js,ts,jsx,tsx}'",
-    "formatCheck": "prettier --check --config .prettierrc './{app,lib}/src/**/*.{js,ts,jsx,tsx}'",
-    "lint:interactive": "yarn eslint-interactive ./app/src ./lib/src",
-    "test": "yarn workspace @streamlit/lib test && yarn workspace @streamlit/app test",
-    "testcoverage": "yarn workspace @streamlit/lib test --coverage && yarn workspace @streamlit/app test --coverage",
-    "testLib": "yarn workspace @streamlit/lib testWatch",
-    "testApp": "yarn workspace @streamlit/app testWatch"
```

Comment:
Are these scripts mostly removed as a cleanup, or is there also a technical issue related to this?

Reply:
Yes. I thought it would be better to just use the syntax for people and thought this would be a reset. Especially cause you can run tests on all workspaces etc.

Reply:
yeah, I think I haven't been using these commands in a longer time. Previously, I sometimes used `testLib` and `testApp` form the main repo but since migrating to vite I have been running all tests just in the VS Code test explorer.